support/Pipelines/Bbh/EccentricityControl.py

```diff
@@ -82,42 +81,17 @@ def eccentricity_control(
 
     # Find the current eccentricity and determine new parameters to put into
     # generate-id
-    (
-        eccentricity,
-        ecc_std_dev,
-        new_orbital_params,
-    ) = eccentricity_control_params(
+    ecc_params = eccentricity_control_params(
         h5_files,
         id_input_file_path,
         tmin=tmin,
         tmax=tmax,
         plot_output_dir=plot_output_dir,
+        ecc_params_output_file=ecc_params_output_file,
     )
 
-    # Create DataFrame to display data in tabular format
-    data = {
-        "Attribute": [
-            "Eccentricity",
-            "Eccentricity error",
-            "Updated Omega0",
-            "Updated adot0",
-        ],
-        "Value": [
-            eccentricity,
-            ecc_std_dev,
-            new_orbital_params["Omega0"],
-            new_orbital_params["adot0"],
-        ],
-    }
-    df = pd.DataFrame(data)
-    # Print header line
-    print("=" * 40)
-    # Display table
-    print(df.to_string(index=False))
-    print("=" * 40)
-
     # Stop eccentricity control if eccentricity is below threshold
-    if eccentricity < 0.001:
+    if ecc_params["Eccentricity"] < 0.001:
```

Comment:
just a quick glance because I'll adapt a lot of this to BNS at some point. Should the threshold be specifiable somewhere? In SpEC we've had a harder time getting lower eccs for high q & chi runs. So much so that sometimes in BFI we just override the BFI-based target ecc.

Reply:
Yes definitely, we can just make it a target parameter, that's very easy to do